### PR TITLE
Fix bug where name could not differentiate whitespace

### LINKED ISSUE #94: Fix whitespace bug in add command
- Add command allows multiple whitespace between names

### LINKED ISSUE #133: View invalid command message whitespace bug
image

Small bug with error message, no space between applicantspecified - should be applicants specified

### PR DESCRIPTION
**Pull request by goel-a**
Bug Fixes:
resolves #94 , resolves #133

### CODE REVIEW

File: src/test/java/seedu/linkedout/model/applicant/ApplicantTest.java [hunk 1]
```diff
@@ -45,10 +45,10 @@ public void isSameApplicant() {
         Applicant editedBob = new ApplicantBuilder(BOB).withName(VALID_NAME_BOB.toLowerCase()).build();
         assertFalse(BOB.isSameApplicant(editedBob));
 
-        // name has trailing spaces, all other attributes same -> returns false
-        String nameWithTrailingSpaces = VALID_NAME_BOB + " ";
-        editedBob = new ApplicantBuilder(BOB).withName(nameWithTrailingSpaces).build();
-        assertFalse(BOB.isSameApplicant(editedBob));
+        // name has multiple whitespaces, all other attributes same -> returns true
+        String nameWithMultipleSpaces = VALID_NAME_BOB + " ";
```
Comment (zzhikai) [inline src/test/java/seedu/linkedout/model/applicant/ApplicantTest.java:49]: This part seems to be single space instead of multiple space correct me if i'm wrong

Comment (goel-a) [reply src/test/java/seedu/linkedout/model/applicant/ApplicantTest.java:49]: The implementation removes all 'extra' whitespaces, be it before, between or after the words

File: src/main/java/seedu/linkedout/logic/commands/ViewCommand.java [hunk 2]
```diff
@@ -14,8 +14,9 @@
 
     public static final String COMMAND_WORD = "view";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": View an overview of a specific applicant "
-            + "specified by applicant's name (case-insensitive) and displays the applicant's information.\n"
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": View an overview of a specific applicant"
```
Comment (ryantanlien) [inline src/main/java/seedu/linkedout/logic/commands/ViewCommand.java:18]: Delete extra whitespace here on line 17

File: src/test/java/seedu/linkedout/model/applicant/ApplicantTest.java (hunk 1 above)
Comment (ryantanlien) [reply src/test/java/seedu/linkedout/model/applicant/ApplicantTest.java:49]: This does not seem to be multiple spaces, instead this is just the trailing whitespace but instead they are now the expected result is true. 

Try to add a second, more comprehensive test (include a scenario where there is a multiple whitespace) Right now `VALID_NAME_BOB = Bob Choo`, try a version with `INVALID_NAME_BOB = VALID_NAME_BOB.replaceAll(" ", "  ")`

**Review by ryantanlien — CHANGES_REQUESTED** [review]

**Review by ryantanlien — APPROVED** [review]
LGTM!
